### PR TITLE
Candidate-app-28: Candidates status

### PR DESCRIPTION
**Pull request by equ1rix**

### CODE REVIEW

**Comment by equ1rix** [comment]
image
image

**Comment by equ1rix** [comment]
image

**Comment by alexdplua** [comment]
Fetch данных сделать на homepage

File: src/components/CandidatesModal/index.tsx [hunk 1]
```diff
@@ -33,11 +35,13 @@ const CandidatesModal = ({ onClose = mock }: CandidatesModalProps) => {
   const [phoneNumber, setPhoneNumber] = useState<string>('');
   const [gitHub, setGitHub] = useState<string>('');
   const [linkedIn, setLinkedIn] = useState<string>('');
-  const [status, setStatus] = useState<boolean>(true);
+  const [status, setStatus] = useState<string>('');
+  const [position, setPosition] = useState('');
```
Comment (alexdplua) [inline src/components/CandidatesModal/index.tsx]: ts

File: src/components/CandidatesModal/index.tsx [hunk 2]
```diff
@@ -33,11 +35,13 @@ const CandidatesModal = ({ onClose = mock }: CandidatesModalProps) => {
...
   const [isFavorite, setIsFavorite] = useState<boolean>(false);
   const { closeModal, isOpenModal } = useContext(ModalContext);
-  const [position, setPosition] = useState('');
 
+  const { statuses } = useFetchStatuses();
+  const { refetchCandidates } = useFetchCandidates(1, '', '', '');
```
Comment (alexdplua) [inline src/components/CandidatesModal/index.tsx]: fix this

**Pull request closed without merging**